modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/ManagementToolbar.soy

```diff
@@ -200,7 +200,9 @@
 			{let $searchData: $customData.searchData /}
 
 			{foreach $key in keys($searchData)}
-				<input name="{$key}" type="hidden" value="{$searchData[$key]}" />
+				{foreach $value in $searchData[$key]}
```

Comment:
Hey @cgoncas, can you provide a pseudo example of what `searchData` looks like after this? I guess it's an array of arrays? Does this continue to work well when that's not the case?

@markocikos, @julien, @ambrinchaudhary, @jonmak08, if we end up adding this, we might want to check our new version to make sure we have support for it from the start.

Reply:
Hey @balazssk!!!

Could you provide the pseudo example requested by @jbalsas?

Thanks!!

Reply:
hey, @jbalsas , @cgoncas 
searchData was a simple map of  key->values, this change modifies it into a map of key->array/list.

I'm not 100% sure how the translation between java, soy and js happens, but from my understanding where previously there was something like a 

`{
   "key1" : "value1",
   "key2" : "value2"
}`
now there is a 
`{
   "key1" : ["value1a", "value1b"],
   "key2" : ["value2"]
}`

I think using your array of arrays analogy, this is more like an object of arrays... maybe?

Reply:
Hey @balazssk, thanks, that's what I guessed, yeah.

I was wondering if these could make other usages fail if not properly updated, but seeing as it is an internal thing it looks okay.

Do you think you could attempt to apply this too to our [new tag implementation](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/ManagementToolbarTag.java#L1064) and its [react component](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/SearchControls.js#L69-L77)?

Otherwise, I think someone in the team should be able to do so, but I'd rather get it all at once to avoid this slipping through the cracks and re-surfacing later in the future.

Reply:
Hi @jbalsas ,
I think I have the solution for the new tag.
I've sent Cristina a followup [PR|https://github.com/cgoncas/liferay-portal/pull/559] so that when she merges it we can have all the code in this one.

I'm not sure how to test it though. Only blogs admin uses the new tag, but search is limited there.

Reply:
I have updated this branch with your changes in the PR

Reply:
@markocikos @jonmak08 @ambrinchaudhary fyi